src/models/bundle-descriptor-constraints.ts

```diff
@@ -149,16 +150,26 @@ export const API_CLAIMS_CONSTRAINTS: UnionTypeConstraints<
         type: 'string'
       },
       bundle: {
-        required: true,
+        required: false,
         type: 'string',
         validators: [bundleRegExpValidator]
+      },
+      bundleReference: {
+        required: false,
+        type: 'string'
       }
     }
   ],
   validators: [
-    mutualDependency(
-      { key: 'type', value: ApiType.External },
-      { key: 'bundle' }
+    mutualExclusive(
```

Comment:
it would be nice to have something like

```
 exclusive('bundle', 'bundleReference'),
 fieldDependsOnFields(
      { key: 'type', value: "external" },
      { key: 'bundleReference' },
      { key: 'bundle'})
```